scripts/kconfig/kconfigfunctions.py

```diff
@@ -976,4 +1059,11 @@ def substring(kconf, _, string, start, stop=None):
         "normalize_upper": (normalize_upper, 1, 1),
         "shields_list_contains": (shields_list_contains, 1, 1),
         "substring": (substring, 2, 3),
+        "add": (add, 1, 255),
+        "sub": (sub, 1, 255),
+        "mul": (mul, 1, 255),
+        "div": (div, 1, 255),
+        "mod": (mod, 1, 255),
```

Comment:
A minor optimization for my OCD, we can have a single function here:

```python
def arith(kconf, name, *args):
    argarray = args if len(args) > 1 else args[0].split(",")

    if name == 'add':
        op = lambda a, b: int(a) + int(b)
    elif name == 'sub':
        op = lambda a, b: int(a) - int(b)
    elif name == 'mul':
        op = lambda a, b: int(a) * int(b)
    elif name == 'div':
        op = lambda a, b: int(a) / int(b)
    elif name == 'mod':
        op = lambda a, b: int(a) % int(b)
    else:
        assert False

    return str(int(functools.reduce(op, argarray)))
```

Reply:
flake8 says,
> E731 do not assign a lambda expression, use a def

The lines aren't that long, so we're trying to complete the processing on each line.

Reply:
I guess you could use the [operator library](https://docs.python.org/3/library/operator.html) 😛 , it's fine though

Reply:
OKay, I applied it.